recipes/python-cityhash/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "python-cityhash" %}
+{% set version = "0.4.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/cityhash/cityhash-{{ version }}.tar.gz
+  sha256: 087c1c9f3d0ca3303bbcf683295151304274059d1efba07ee8429b3b80c33af0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - cython
+    - py-cpuinfo
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - cityhash
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/escherba/python-cityhash
+  summary: Python bindings for CityHash and FarmHash
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - charlesbluca
+    - escherba
```

Comment:
Going to go ahead and remove for now to avoid potentially unwanted notifications


```suggestion
```